src/main/java/com/thebois/listeners/events/ObstaclePlacedEvent.java

```diff
@@ -0,0 +1,36 @@
+package com.thebois.listeners.events;
+
+import com.thebois.models.Position;
+
+/**
+ * Describes an obstacle that has been placed in the world.
+ */
+public class ObstaclePlacedEvent {
+
+    private final int posX;
+    private final int posY;
```

Comment:
`Position` uses floats, and using it would mean having to cast the values to `int` at multiple places. This way, only valid obstacle positions make it into the event. The getter `getPosition` still returns a `Position` to make using the event easier.